Misc/NEWS.d/next/Library/2017-11-01-03-28-24.bpo-31917.DYQL0g.rst

```diff
@@ -0,0 +1 @@
+Add time.CLOCK_PROF constant.
```

Comment:
Perhaps mention that this FreeBSD-only here?

Reply:
I added an entry in What's New in Python 3.7 which adds more details.